packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -186,22 +189,21 @@ const ActionHeader = ({
             />
           )}
         </FormattedMessage>
-
       </div>
     )
     : null;
 
   const leftButtons = (backButton || expandButton || previousNextButtonGroup)
     ? (
-      <div className={cx('left-buttons')}>
+      <div className={cx('left-buttons', theme.className)}>
```

Comment:
I've been wondering if we should add jest tests for adding these contexts.

Reply:
added jest test: https://github.com/cerner/terra-core/pull/2921/commits/3d1b58a89c60a3b30efbe68fc2bf6c4919c75074